modules/payloads/stagers/java/reverse_http.rb

```diff
@@ -53,8 +53,8 @@ def config
     c << "Spawn=#{spawn}\n"
     c << "URL=http://#{datastore["LHOST"]}"
     c << ":#{datastore["LPORT"]}" if datastore["LPORT"]
-    c << "/"
-    c << generate_uri_checksum(Rex::Payloads::Meterpreter::UriChecksum::URI_CHECKSUM_INITJ, uri_req_len)
```

Comment:
I think this should be: ```c << generate_uri_uuid_mode(:init_java, uri_req_len)``` like it is in java/reverse_https.rb

Rspec is hanging because we're getting stuck trying to call generate_uri_checksum for a 5 byte uri_req_len. That's because in the context of rspec, we don't know the available space, so it's defaulting to 5 bytes. I'm not sure what that is there. It seems arbitrary to me.